CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Default stackableVersion to operator version ([#493]).
```

Comment:
I'd say this needs a more prominent warning that this is a CRD change?

Reply:
Do you have any suggestion of what it could look like? Its non-breaking btw ;)

Reply:
I have not checked if it's valid Markdown but something along those lines.

Reply:
WDYT of

- Default stackableVersion to operator version. It is recommended to remove `spec.image.stackableVersion` from your custom resources ([#493]).

Reply:
As you are on vacation, and @siegfriedweber needs this PR to continue his work I would go with my proposed entry. We can continue discussing it here, after we reach a conclusion we need to change it in *all* operators anyway (so one operator more does not make a big change)

Reply:
I need to resolve this for merging, reopening it afterwards